apps/bsatool/bsatool.cpp

```diff
@@ -74,69 +71,69 @@ bool parseOptions (int argc, char** argv, Arguments &info)
     try
     {
         bpo::parsed_options valid_opts = bpo::command_line_parser(argc, argv)
-            .options(all).positional(p).run();
+                .options(all).positional(p).run();
         bpo::store(valid_opts, variables);
     }
-    catch(std::exception &e)
+    catch (std::exception& e)
     {
         std::cout << "ERROR parsing arguments: " << e.what() << "\n\n"
-            << desc << std::endl;
+                  << desc << std::endl;
         return false;
     }
 
     bpo::notify(variables);
 
-    if (variables.count ("help"))
+    if (variables.count("help"))
```

Comment:
Another case of not entirely consistent style. From what I remember the style with an additional space here is more common. I could be wrong about that though.

Reply:
Yes, this is not consistent throughout the code base, sometimes with or without a space.

Perhaps we should talk about this in the forum and come up with a .clangformat that everyone can use?

Reply:
https://forum.openmw.org/viewtopic.php?f=42&t=6790  <-- for you @zinnschlag :D
(and @Capostrophic @akortunov )

Reply:
Allman doesn't address spaces all that much though.

Reply:
[I like spaces!](https://upload.wikimedia.org/wikipedia/en/thumb/1/14/Ralph_Wiggum.png/220px-Ralph_Wiggum.png)